examples/disclosure/disclosure-navigation-hybrid.html

```diff
@@ -0,0 +1,378 @@
+    <li><a href="disclosure-faq.html">Example Disclosure (Show/Hide) for Answers to Frequently Asked Questions</a></li>
+    <li><a href="disclosure-img-long-description.html">Example Disclosure (Show/Hide) for an Image Description</a></li>
+    <li><a href="../menubar/menubar-navigation.html">Navigation Menubar Example</a></li>
+  </ul>
+  <h2>Example Usage Options</h2>
+  <p>
+    This example demonstrates two different ways of implementing keyboard support.
+    Toggle between them with the following checkbox.
+  </p>
+  <label for="arrow-behavior-switch">
+    <input type="checkbox" id="arrow-behavior-switch" checked>
+    Include optional support for arrow keys, <kbd>Home</kbd>, and <kbd>End</kbd>
+  </label>
+
+  <section>
+    <div class="example-header">
+      <h2 id="ex_label">Example</h2>
+    </div>
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of" ></div>
+    <div id="ex1">
+      <nav aria-label="Mythical University">
+        <ul id="exTest" class="disclosure-nav">
+          <li>
+            <a href="#mythical-page-content" class="main-link">About</a>
+            <button type="button" aria-expanded="true" aria-controls="id_about_menu" aria-label="Show About section"> </button>
+            <ul id="id_about_menu">
+              <li>
+                <a href="#mythical-page-content">Overview</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Administration</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Facts</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Campus Tours</a>
+              </li>
+            </ul>
+          </li>
+          <li>
+            <a href="#mythical-page-content" class="main-link">Admissions</a>
+            <button type="button" aria-expanded="true" aria-controls="id_admissions_menu" aria-label="Show Admissions section"> </button>
+            <ul id="id_admissions_menu">
+              <li>
+                <a href="#mythical-page-content">Apply</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Tuition</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Sign Up</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Visit</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Photo Tour</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Connect</a>
+              </li>
+            </ul>
+          </li>
+          <li>
+            <a href="#mythical-page-content" class="main-link">Academics</a>
+            <button type="button" aria-expanded="true" aria-controls="id_academics_menu" aria-label="Show Academics section"> </button>
+            <ul id="id_academics_menu">
```

Comment:
Prevent initial flash.
```suggestion
            <button type="button" aria-expanded="false" aria-controls="id_academics_menu" aria-label="Show Academics section"> </button>
            <ul id="id_academics_menu" style="display: none;">
```

Reply:
I thought we did this so that if JS weren't enabled or didn't load, it would still work. Is that not a concern we're targeting?